citesphere/src/main/java/edu/asu/diging/citesphere/core/util/model/impl/GilesUtil.java

```diff
@@ -0,0 +1,44 @@
+package edu.asu.diging.citesphere.core.util.model.impl;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+import edu.asu.diging.citesphere.core.util.model.IGilesUtil;
+import edu.asu.diging.citesphere.model.bib.IGilesUpload;
+
+@Component
+public class GilesUtil implements IGilesUtil {
+
+    private Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Override
+    public void convertFilesToBytesList(List<byte[]> fileBytes, MultipartFile[] files) {
```

Comment:
`MultipartFile`s are strictly front end, so that especially shouldn't be in the model package. It should probably not even be outside the web package.

Reply:
why does this method ask for a list of byte arrays? Do you expect this method to be called for a list that already contains objects? if not, just create the list object here.